docker/data-science/notebooks/ch06/2.related-keywords-from-signals.ipynb

```diff
@@ -163,12 +165,12 @@
       "|    iphone 4s|         iphone|        21|\n",
       "|       laptop|      hp laptop|        20|\n",
       "|         thor|captain america|        18|\n",
-      "|         bose|          beats|        17|\n",
       "|   skullcandy|          beats|        17|\n",
       "|    iphone 4s|       iphone 4|        17|\n",
+      "|         bose|          beats|        17|\n",
```

Comment:
Note: Sort order changed for some reason. The manuscript has the previous version still. Not sure why this sorting is non-deterministic.